### PR TITLE
PhysicalMachineChaos: add validate for network bandwidth

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Signed-off-by: xiang <xiang13225080@163.com>

### What problem does this PR solve?

fix issue: https://github.com/chaos-mesh/chaos-mesh/issues/2813

### What's changed and how it works?

add validate for network bandwidth

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [x] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

File: api/v1alpha1/physical_machine_chaos_webhook.go [hunk 1]
```diff
@@ -131,8 +133,6 @@ func (in *PhysicalMachineChaosSpec) Validate(root interface{}, path *field.Path)
 	case PMClockAction:
 		validateConfigErr = validateClockAction(in.Clock)
 	default:
-		allErrs = append(allErrs,
```
Comment (WangXiangUSTC) [inline api/v1alpha1/physical_machine_chaos_webhook.go:134]: action already checked in https://github.com/chaos-mesh/chaos-mesh/blob/80c4d00a302de1f1d2dca609395b05d2c4db1fdc/api/v1alpha1/physical_machine_chaos_types.go#L67, so removed it

**Review by iguoyr — APPROVED** [review]
LGTM

**Review by Andrewmatilde — APPROVED** [review]
LGTM

**Pull request merged**